include/grass/defs/imagery.h

```diff
@@ -160,7 +160,7 @@ void I_free_signatures_list(int, char ***);
 int I_init_signatures(struct Signature *, int);
 int I_new_signature(struct Signature *);
 int I_free_signatures(struct Signature *);
-int I_read_one_signature(FILE *, struct Signature *);
+int I_read_one_signature(FILE *, int, struct Signature *);
```

Comment:
Seems like an API change. Probably not serious in practice since nobody picked up the new signature API yet. It still should be clarified in the doc or something.

Reply:
I_read_one_signature should not be a part of public API as it is just an implementation detail of I_read_signatures.

Reply:
If the function cannot be static in a file, than the method used elsewhere to mark it as internal/private/protected is double underscore after the prefix, i.e., `I__read_one_signature`. I think that works pretty well, although it is probably not well documented. Unfortunately, Doxygen does not have a good way of marking internal functions, `@protected` and `@private` might be somewhat close to the right thing.

Reply:
I'm not so keen on making large changes as imagery code predates 3.2... I just added a docstring with a note.